app/resources/css-default.template

```diff
@@ -1206,12 +1194,12 @@ table.api-keys td.empty {
 }
 
 .overlay {
-  position:fixed;
-  top:0;
-  left:0;
-  height: 100%;
-  width: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
   z-index: 2;
   background: rgba(0,0,0,0.5);
-  overflow-x: hidden;
+  display: none;
 }
```

Comment:
Looks like you are still missing the line break. The red circle on Github indicates that the line misses a line break.